src/module/rules/rule-element/item-alteration/alteration.ts

```diff
@@ -194,6 +195,23 @@ class ItemAlteration extends foundry.abstract.DataModel<RuleElementPF2e, ItemAlt
 
                 return;
             }
+            case "damage-dice-number": {
+                const validator = ITEM_ALTERATION_VALIDATORS[this.property];
+                if (!validator.isValid(data) || !(data.item instanceof ItemPF2e)) {
```

Comment:
I think this should be safe to run on source data. Are you getting an error without the `Item` instance check?

Reply:
Ah, nope, seems like removing the Item instance check causes no compilation errors.